var/spack/repos/builtin/packages/papi/package.py

```diff
@@ -113,6 +114,10 @@ def setup_build_environment(self, env):
 
     setup_run_environment = setup_build_environment
 
+    def autoreconf(self, spec, prefix):
+        bash = which("bash")
+        bash("-c", "cd src && autoreconf -ivf")
+
```

Comment:
@hppritcha Can you check why this was necessary (per @G-Ragghianti request)?

Reply:
probably config.guess needed to be updated?   Its been too long to remember exactly why this was needed.

Reply:
oh duh the brain cell that worked on this PR now remembers.  the patch patches configure.in so autoreconf needs to be run to generate the new, correct, configure.

Reply:
oh and the autotools stuff was all in src (like configure.in) not in the top level folder.